grafana_sync.py

```diff
@@ -79,6 +79,9 @@ def set_dashboard(base_url, session, data_json, overwrite=False, adapt_uid=False
         raise ValueError("JSON contains no dashboard")
 
     dashboard_json = data_json
+    #Ensure is a createion
+    data_json['dashboard'].pop('id',None)
```

Comment:
It was not working without this changes

Reply:
Good catch, I did make the change. 

This part of the code is a little weak. On alert, I did find an issue when copying from an environment with manually copied alerts and trying to use the tool. Then it found duplicate alerts as the same title is used but a different id.

I did not work on that, and here more or less we will have the same problem. At the moment I did not put effort here. We need to see how the tool will be used, and then make it grow in that direction.

Reply:
OK